static/app/components/events/eventTagsAndScreenshot/index.tsx

```diff
@@ -24,7 +23,9 @@ export function EventTagsAndScreenshot({projectSlug, event, isShare = false}: Pr
     {enabled: !isShare}
   );
   const screenshots =
-    attachments?.filter(({name}) => SCREENSHOT_NAMES.includes(name)) ?? [];
+    attachments?.filter(
+      ({name}) => name.endsWith('screenshot.jpg') || name.endsWith('screenshot.png')
```

Comment:
do we know if "screenshot-1" is still a case we want to cover?

Reply:
I will expand the test case on Monday. There's no one with certainty as to what they want.